ShanghaiBloodSports/Assets/Scripts/MockInputBuffer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
```

Comment:
I really like the idea of mocking, but does unity support subfolders?
e.g. Scripts/InputBuffer/MockInputBuffer.cs

We're starting to get big enough that I think we should look into organizing the scripts a bit more

Reply:
I agree, let's do that in another pull request